src/reducers/index.js

```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return {
+        count: action.payload
```

Comment:
If you're not sending the `count` in action payload, you can still update the state by doing:
```
  return {
    count: state + 1
  }
```